x/pairing/keeper/pairing.go

```diff
@@ -153,6 +179,47 @@ func (k Keeper) getPairingForClient(ctx sdk.Context, chainID string, clientAddre
 	return providers, vrfk, allowedCU, legacyStake, err
 }
 
+func minMaxProvidersToPair(values []uint64) (uint64, error) {
```

Comment:
no need for error.
look at this implementation: 
https://stackoverflow.com/questions/27516387/what-is-the-correct-way-to-find-the-min-between-two-integers-in-go

Reply:
The error is only for a situation where all the policies' MaxProvidersToPair values are invalid (smaller than 2). You can see in the function that I've initialized the `min` value to be `math.MaxUint64`. The `min` value won't change if all of the MaxProvidersToPair values are invalid. Only in this case I return an error.

Should I omit this logic?

Reply:
i think that the check for the min provider should be at the assigning of the policy, set policy or create new project or adding a new plan, no need to check it all over again